src/views/Home.js

```diff
@@ -50,27 +50,47 @@ export class Home extends Component {
         );
     }
 
+    _reloadComponent() {
+        this.props.setAuthToken(undefined);
+        this._renderButtons()
```

Comment:
This line won't do anything ;)

Reply:
Ohhh because it's always refreshing anyways right?